spec.md

```diff
@@ -1616,40 +1614,49 @@ For success responses, the following fields are defined:
 }
 ```
 
-## Orphans
+## Orphan Mitigation
 
 The Platform is the source of truth for Service Instances and Service Bindings.
 Service Brokers are expected to have successfully provisioned all of the Service
 Instances and Service Bindings that the Platform knows about, and none that it
 doesn't.
 
-Orphans can result if the Service Broker does not return a response before a
-request from the Platform times out (typically 60 seconds). For example, if a
-Service Broker does not return a response to a provision request before the
-request times out, the Service Broker might eventually succeed in provisioning
-a Service Instance after the Platform considers the request a failure. This
-results in an orphan Service Instance on the Service Broker's side.
+Orphaned Service Instances and Service Bindings  MAY be created by in any of the
+following scenarios:
+* The Service Broker does not return a response before a request from the
+Platform times out (typically 60 seconds). The Service Broker might eventually
+succeed in creating a resource, however the Platform MAY have already considered
```

Comment:
s/may/might/

Reply:
I think 'might' is better because I don't think the sentence (and the MAY) is meant to be using RFC2119 keywords - the sentence is non-normative - it's more just talking about a situation that might happen - it's not saying that the platform should (or should not) actually do something per the spec.

Reply:
Did you really want to keep the MAY here?

Reply:
Nope, fixed